### PR TITLE
Updates README to note that Pootle is not actively maintained

### LINKED ISSUE #6920: Status of that project?
The last commit was in June 2022 (last year).

Is the project still maintained somehow?

### PR DESCRIPTION
**Pull request by linghengqian**
- Fixes #6920 .
- Updates README to note that Pootle is not actively maintained. See https://github.com/translate/pootle/issues/6920#issuecomment-1525567199 and https://github.com/translate/pootle/issues/6920#issuecomment-1525596832 .

### CODE REVIEW

**Review by linghengqian — COMMENTED** [review]
- CC @phlax .

File: README.rst [hunk 1]
```diff
@@ -1,6 +1,9 @@
 |logo| Pootle
 =============
 
+Pootle is in the process of looking for potential maintainers to take over. Until then, users can use `Weblate <https://github.com/WeblateOrg/weblate>`_ as an alternative.
+------------------
```
Comment (phlax) [inline README.rst:5]: in rst the underline needs to be at least as long as the sentence - usually best to make them the same length

**Review by phlax — COMMENTED** [review]
thanks for this @linghengqian - not 100% about status of docs publishing but would be good to land this i think

probably we should add something like

> If you are interested in maintaining Pootle please open an issue at ...

File: README.rst (hunk 1 above)
Comment (linghengqian) [reply README.rst]: - I modified the document. To be honest I had never used reStructuredText before.

**Review by phlax — COMMENTED** [review]
how about not making these headers?

perhaps add a header like

```rst

Maintainers needed
---------------------------

...
```

and then add the points you have as the body of the paragraph

**Comment by linghengqian** [comment]
> and then add the points you have as the body of the paragraph

- How about this? I updated the commit.
- image

**Review by phlax — APPROVED** [review]
lgtm, thanks @linghengqian

**Pull request merged**